### ISSUE TITLE
About dataset splits.

### ISSUE THREAD

**Issue by jaewonchu** [HKUDS/LightGCL#17]
Hi. I am a very interested reader of your paper. I have a question about dataset splits and would be grateful if you answer me when you have time.

In the paper, each dataset has the following interactions:

Yelp: 1,517,326
Gowalla: 1,172,425
ML-10M: 9,988,816
Amazon-book: 2,240,156
Tmall: 2,357,450

and all datasets are split into training, validation, and testing with a ratio of 7:2:1, which means each training, validation, and testing dataset has the following interactions:

Yelp: 1,062,128 / 303,465 / 151,733 (70%, 20%, 10%)
Gowalla: 820,697 / 234,485 / 117,243 (70%, 20%, 10%)
ML-10M: 6,992,171 / 1,997,763 / 998,882 (70%, 20%, 10%)
Amazon-book: 1,568,109 / 448,031 / 224,016 (70%, 20%, 10%)
Tmall: 1,650,215 / 471,490 / 235,745 (70%, 20%, 10%)

But I found that the uploaded dataset has the following interactions: (trnMat.pkl and tstMat.pkl)

Yelp: 1069128 / - / 305466 (70%, -, 20%)
Gowalla: 1172425 / - / 130270 (100%, -, 10%)
ML-10M: 6999171 / - / 1999761 (70%, -, 20%)
Amazon-book: 2240156 / - / 640045 (100%, -, 30%)
Tmall: 2357450 / - / 261939 (100%, -, 10%)

which is far different from the paper. Am I missing something important? 
I look forward to hearing back from you.

**Comment by akaxlh** [comment]
Hi, thank you for pointing out this issue. There seems to be some differences in the data split for different datasets, but we only reported one split protocol in the paper. Specifically, Yelp, Movielens, and Amazon were split by 7:1:2, but Gowalla and Tmall were split by 7:2:1. 
Also, there are two other points regarding this issue:
1.	The statistic table in paper may be inconsistent across different datasets, for some of which the total number of interactions is reported, while for some of which only the number of training interactions is reported. This may be why you get 100%.
2.	For some datasets, the uploaded training set is actually the fusion of original training set and the validation set. The validation set was used to find best hyperparameter set, including number of training epochs. For some datasets, we finally fused the training set and validation set. And this fusion was done for all the compared methods.
